addon/components/flexberry-tab-bar.js

```diff
@@ -0,0 +1,140 @@
+import Ember from 'ember';
+import layout from '../templates/components/flexberry-tab-bar';
+
+/**
+  Component's CSS-classes names.
+  JSON-object containing string constants with CSS-classes names related to component's .hbs markup elements.
+
+  @property {Object} flexberryClassNames
+  @property {String} flexberryClassNames.prefix Component's CSS-class names prefix ('flexberry-tab-bar').
+  @property {String} flexberryClassNames.wrapper Component's wrapping <div> CSS-class name ('flexberry-tab-bar').
+  @property {String} flexberryClassNames.tab Component's inner <input type="checkbox"> CSS-class name ('flexberry-tab-bar-tab').
+  @property {String} flexberryClassNames.tabIcon Component's inner <label> CSS-class name ('flexberry-tab-bar-tab-icon').
+  @readonly
+  @static
+
+  @for FlexberryTabBarComponent
+*/
+const flexberryClassNamesPrefix = 'flexberry-tab-bar';
+const flexberryClassNames = {
+  prefix: flexberryClassNamesPrefix,
+  wrapper: flexberryClassNamesPrefix,
+  tab: flexberryClassNamesPrefix + '-tab',
+  tabIcon: flexberryClassNamesPrefix + '-tab-icon'
+};
+
+/**
+ * FlexberryTabBarComponent
+ * Component to display semantic ui tabs
+ * @extends Ember.Component
+ */
+export default Ember.Component.extend({
+  classNames: ['flexberry-tab-bar'],
```

Comment:
It's better to use previously defined constant 'flexberryClassNamesPrefix' here.